src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -359,10 +359,12 @@ def help_text
       # @return [Boolean] true when skipping has been confirmed
       def show_skipping_warning
         # Popup question: confirm skipping the registration
-        warning = _("If you do not register your system we will not be able\n" \
-            "to grant you access to the update repositories.\n\n" \
-            "You can register after the installation or visit our\n" \
-            "Customer Center for online registration.")
+        warning = _("Without registration, update-channels will not be\n" \
+            "configured. This disables updates and security fixes.\n\n" \
+            "A full system can be installed using the\n" \
+            "SLE-15-Packages media from download.suse.com.\n" \
```

Comment:
"media" is a plural. The singular is "medium".
Is the medium name correct?
How stable is it? Should we insert it with `format`?

Reply:
The medium name is correct. However I would not hard code it here but rather use a string interpolation `%{media_name}` placeholder as it still can be changed. The same with the site name.

That allows us to update the names without breaking the translations.

BTW I'd probably prefer `https://download.suse.com` value so it's clear what does it mean.